xrdp/xrdp_painter.c

```diff
@@ -76,6 +77,10 @@ xrdp_painter_add_dirty_rect(struct xrdp_painter *self, int x, int y,
     return 0;
 }
 
+#define RFX_TILE_SQUARE 64
+
+#define XRDP_SURCMD_PREFIX_BYTES 256
```

Comment:
Where do these come from?

Reply:
Thank you for looking at my stuff!

I just looked again and I think the first one might want to be replaced by CT_TILE_64x64 from src/rfxconstants.h, assuming that's visible outside the RFX encoder implementation.  (I wonder what "CT" stands for?)

The second, XRDP_SURCMD_PREFIX_BYTES, is a copy of an existing #define in xrdp_encoder.c, where it serves the same purpose (leaving room for an RDP surface command so that the image encoding doesn't have to be copied).  Maybe the #define belongs in libxrdp/libxrdpinc.h?